data/noiseprofiles.json

```diff
@@ -850,6 +850,21 @@
             {"skip": true, "name": "EOS 200D iso 51200", "iso": 51200, "a": [0.00249000518027299, 0.00112357559710744, 0.00181437909489014], "b": [1.72431912704092e-05, 2.76503246829995e-05, 2.03920295934623e-05]}
           ]
         },
+        {
+          "comment": "EOS 200D II (EOS 250D) contributed by scybernaut",
```

Comment:
Could add this is also EOS Rebel SL3? Maybe even better change to "eos 250d (aka eos 200d II, eos rebel sl3) ..."

Reply:
Thanks for the review, I updated the comment accordingly